chrome/content/zotero/xpcom/translation/translators.js

```diff
@@ -418,23 +426,45 @@ Zotero.Translators = new function() {
 			throw new Error("code not provided");
 		}
 		
-		var fileName = Zotero.Translators.getFileNameFromLabel(
-			metadata.label, metadata.translatorID
-		);
-		var destFile = OS.Path.join(Zotero.getTranslatorsDirectory().path, fileName);
-		
 		// JSON.stringify has the benefit of indenting JSON
 		var metadataJSON = JSON.stringify(metadata, null, "\t");
-		
-		var str = metadataJSON + "\n\n" + code,
-			translator;
+		var str = metadataJSON + "\n\n" + code;
 		
 		var translator = Zotero.Translators.get(metadata.translatorID);
-		var sameFile = translator && destFile == translator.path;
-		if (sameFile) return;
```

Comment:
I feel like I'm missing something here, since the statement is so explicit (unless it was just an oversight)